### ISSUE TITLE
Install libraries in /usr/lib64 instead of /usr/lib

### ISSUE THREAD

**Issue by TheChymera** [ANTsX/ANTs#872]
**Describe the bug**
I would like to build ANTs so that the libraries get installed into `usr/lib64` instead of `usr/lib` . I believe this should be configurable via the CMake build system.

**To Reproduce**
I am building ANTs with:

```
cmake -C /var/tmp/portage/sci-biology/ants-9999/work/ants-9999_build/gentoo_common_config.cmake -G Unix Makefiles -DCMAKE_INSTALL_PREFIX=/usr -DUSE_SYSTEM_ITK=ON -DITK_DIR=/usr/include/ITK-5.0/ -DBUILD_TESTING=OFF -DUSE_VTK=OFF -DUSE_SYSTEM_VTK=OFF -DCMAKE_BUILD_TYPE=Gentoo -DCMAKE_TOOLCHAIN_FILE=/var/tmp/portage/sci-biology/ants-9999/work/ants-9999_build/gentoo_toolchain.cmake  /var/tmp/portage/sci-biology/ants-9999/work/ants-9999
```

This is the full [build.log](https://github.com/ANTsX/ANTs/files/3714834/build.log).

**Comment by gdevenyi** [comment]
PRs welcome: https://rix0r.nl/blog/2015/08/13/cmake-guide/

**Comment by TheChymera** [comment]
@gdevenyi is the build 64-bit only?

**Comment by TheChymera** [comment]
Similar to re-report in https://github.com/ANTsX/ANTs/issues/1061

Fixed by the pending PR: https://github.com/ANTsX/ANTs/commit/329a7be631b33294d9cf3d58e8cdeb62e0aa993c

**Comment by cookpa** [comment]
Fixed in #1062
